scripts/util.ts

```diff
@@ -88,3 +103,108 @@ export async function addTokens(
   // Return token information
   return tokens;
 }
+
+export async function deleteOrders(
+  orderIds: number[],
+  accounts: string[],
+  exchange: BatchExchangeInstance,
+): Promise<void> {
+  log.info(`Canceling ${orderIds.length} orders for ${accounts.length} users`);
+  for (let i = 0; i < orderIds.length; i++) {
+    const orderId = orderIds[i];
+    const account = accounts[i];
+    const cancelReceipt = await exchange.cancelOrders([orderId], {
+      from: account,
+    });
+    const events = cancelReceipt.logs.map((log: any) => log.event).join(", ");
+    log.info(
+      `Canceled/Deleted order ${orderId} for user {${account}}. Emitted events: ${events}`,
+    );
+  }
+}
+
+export async function submitSolution(
+  name: string,
+  batchId: number,
+  solution: SolutionSubmission,
+  solverAddress: string,
+  exchange: BatchExchangeInstance,
+): Promise<void> {
+  log.info(`Submit "${name}":
+  - Objective value: ${solution.objectiveValue}
+  - Touched orders: ${solution.touchedorderIds.join(", ")}
+  - Volumes: ${solution.volumes.join(", ")}
+  - Prices: ${solution.prices.join(", ")}
+  - Token ids for prices: ${solution.tokenIdsForPrice.join(", ")}`);
+  const objectiveValue = await exchange.submitSolution(
+    batchId,
+    1,
+    solution.owners,
+    solution.touchedorderIds,
+    solution.volumes,
+    solution.prices,
+    solution.tokenIdsForPrice,
+    { from: solverAddress },
+  );
+  log.info(`Transaction for ${name}: ${objectiveValue.tx}`);
+}
+
+export async function getBatchId(
+  exchange: BatchExchangeInstance,
+): Promise<number> {
+  const batchId = await exchange.getCurrentBatchId();
+  return batchId.toNumber();
+}
+
+export async function createMintableToken(
+  artifacts: Truffle.Artifacts,
+): Promise<Erc20MintableInstance> {
+  const ERC20Mintable = artifacts.require("ERC20Mintable");
+  return ERC20Mintable.new();
+}
+
+async function mintToken(
+  token: Erc20MintableInstance,
+  account: string,
+  amount: string,
+  minter: string,
+) {
+  log.info(
+    `Mint ${amount} of token ${token.address} for user ${account}. Using ${minter} as the minter`,
+  );
+  await token.mint(account, amount, { from: minter });
+}
+
+export async function mintTokens(
+  tokens: Erc20MintableInstance[],
+  users: string[],
+  amount: string,
+  minter: string,
+): Promise<void> {
+  for (let i = 0; i < tokens.length; i++) {
+    for (let j = 0; j < users.length; j++) {
+      await mintToken(tokens[i], users[j], amount, minter);
+    }
+  }
+}
+
+async function _mintOwl(
```

Comment:
ubernit: I'm assuming this name was like this before, I would have preferred something like the following (as I find pre/post fixing identifiers with `_` doesn't really give much information):
```suggestion
async function mintOwlForUser(
```

Reply:
Cool, yea it was like this before.I will make the change, but I think there is another place in the code that needs to be updated so I wont "commit this suggestion"